package.json

```diff
@@ -249,7 +249,7 @@
     "@metamask/gas-fee-controller": "^6.0.1",
     "@metamask/jazzicon": "^2.0.0",
     "@metamask/key-tree": "^9.0.0",
-    "@metamask/keyring-controller": "^7.0.0",
+    "@metamask/keyring-controller": "7.2.0",
```

Comment:
What do you think about using a version range here (7.2.0 - 8.0.0) instead of just targeting 7.2.0?

``` suggestion
    "@metamask/keyring-controller": "^7.2.0",
```

Reply:
@mcmire I haven't paid attention to this fixed